test/CMakeLists.txt

```diff
@@ -1,4 +1,7 @@
+if(WIN32)
+else()
 add_compile_options(-O0 -g -fno-inline-functions -fno-inline)
+endif()
```

Comment:
```
if(NOT WIN32)
```
https://cmake.org/cmake/help/latest/command/if.html

Reply:
Changed.  If later I find I need a Windows-side optimization, for example:
```
add_compile_options( /Od )
```

I might re-add with `if(WIN32)`.  But I don't have a compile option I am using yet so I am using `if(NOT WIN32)` as you suggested.